classes/core/PKPRouter.php

```diff
@@ -178,6 +178,11 @@ public function getRequestedContextPath(PKPRequest $request): string
      */
     public function getContext($request, $forceReload = false)
     {
+        // If the app currently run on CLI, return the on CLI context if set
+        if (runOnCLI()) {
+            return Application::get()->getCliContext();
+        }
```

Comment:
If the app running on CLI, we for sure only have option to retrieve the context information if a cli session based context information is set. So trying to retrieve it from there (e.g. either the context object or null returned) .